cEP-0006.md

```diff
@@ -87,3 +83,71 @@ Original text courtesy of the [Speak Up! project] (http://web.archive.org/web/20
 
 If you have questions, feel free to contact us by emailing `community AT
 coala DOT io`.
+
+
+Reporting a Code of Conduct Violation
+-------------------------------------
+
+If you believe someone is violating the code of conduct we ask that you report
+it by creating an issue at https://gitlab.com/coala-coc/coc-issues, and check
+the "This issue is confidential and should only be visible to team members with
+at least Reporter access." checkbox.  We strive to keep all issues confidential.
+
+In your report please include:
+
+* Your contact info (so we can get in touch with you if we need to follow up)
+* Names (real, nicknames, or pseudonyms) of any individuals involved.
+  If there were other witnesses besides you, please try to include them as well.
+* When and where the incident occurred. Please be as specific as possible.
+* Your account of what occurred. If there is a publicly available record (e.g.
```

Comment:
Maybe add something like "Which part of the CoC specificaly do you think was violated?"? Should be clear in most cases, but just to be sure.